npm/pkg/controlplane/controllers/v2/networkPolicyController_test.go

```diff
@@ -48,7 +48,7 @@ func (f *netPolFixture) newNetPolController(_ chan struct{}, dp dataplane.Generi
 	kubeclient := k8sfake.NewSimpleClientset(f.kubeobjects...)
 	f.kubeInformer = kubeinformers.NewSharedInformerFactory(kubeclient, noResyncPeriodFunc())
 
-	f.netPolController = NewNetworkPolicyController(f.kubeInformer.Networking().V1().NetworkPolicies(), dp)
+	f.netPolController = NewNetworkPolicyController(f.kubeInformer.Networking().V1().NetworkPolicies(), dp, false)
```

Comment:
could we add some NPM Lite tests in this file? Realized we are missing them from https://github.com/Azure/azure-container-networking/pull/3005#discussion_r1762124172

Reply:
Added a unit test here